### PR TITLE
Feat criteria

### PR DESCRIPTION
**Pull request by doktordirk**
as required by current aurelia-auth

  updateMe(body, criteria) {
    if (typeof criteria === 'string' || typeof criteria === 'number') {
      criteria = {id: criteria};
    }
    return this.client.update(this.auth.getProfileUrl(), criteria, body);
  }

### CODE REVIEW

File: doc/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+<a name"2.0.4"></a>
+### 2.0.4 (2016-02-01)
+
+
+#### Bug Fixes
+
+* **rest:** allow criteria as object for update and detroy ([429be2bc](https://github.com/SpoonX/aurelia-api/commit/429be2bc))
```
Comment (RWOverdijk) [inline doc/CHANGELOG.md]: Destroy*

File: build/tasks/server.js [hunk 2]
```diff
@@ -11,7 +11,9 @@ app.all('*', function(req, res) {
     path: req.path,
     query: req.query,
     body: req.body,
-    method: req.method
+    method: req.method,
+    contentType: req.header('content-type'),
+    Authorization: req.header('Authorization')
```
Comment (RWOverdijk) [inline build/tasks/server.js]: I'd perhaps prefer sending back all headers. We'll do that in the next PR that needs to edit this.

**Comment by doktordirk** [comment]
wanted all headers, but i didn't find that in express docs!?

**Comment by RWOverdijk** [comment]
@doktordirk I'm pretty sure that's just req.headers :)

**Comment by doktordirk** [comment]
fyi: babel-eslint 4.1.7 out. "escope": "3.3.0"  can get removed again

**Pull request merged**